inscriptions/ISic030013.xml

```diff
@@ -74,30 +76,30 @@
                             <supportDesc>
                                 <support>
                                     <p></p>
-                                    <material ana="#material.metal.lead" ref="">lead</material> <!-- add ref to eagle vocabulary -->
-                                    <objectType ana="#object.lamina" ref="">lamina</objectType> <!-- add ref to eagle vocabulary -->
+                                    <material ana="#material.metal.lead" ref="https://www.eagle-network.eu/voc/material/lod/108.html">lead</material> 
+                                    <objectType ana="#object.lamina" ref="http://www.eagle-network.eu/voc/objtyp/lod/145.html">lamina</objectType>
                                     <dimensions>
-                                        <height unit="cm"/> <!-- add value -->
-                                        <width unit="cm"/>
+                                        <height unit="cm" atMost="5">5</height> <!-- from Bettarini -->
+                                        <width unit="cm" atMost="12">12</width>
                                         <depth unit="cm"/>
                                     </dimensions>
                                 </support>
-                                <condition/>
+                                <condition ana="#condition.complete"/>
                             </supportDesc>
                             <layoutDesc>
                                 <layout>
-                                    <rs ana="#execution.engraved" ref="">engraved</rs> <!-- add ref to eagle vocabulary -->
-                                    <p></p>
-                                    <damage/>
+                                    <rs ana="#execution.engraved" ref="https://www.eagle-network.eu/voc/writing/lod/3.html">engraved</rs> 
+                                    <p>L'imprecazione è scritta in direzione progressiva, i nomi dei defissi invece sono scritti in direzione regressiva. (Arena)</p>
```

Comment:
The curse is written left-to-right while the names of the victims are written right-to-left.